spec/mongoid/persistable/creatable_spec.rb

```diff
@@ -4,14 +4,14 @@
 
 describe Mongoid::Persistable::Creatable do
 
-  describe ".create" do
+  describe ".create!" do
```

Comment:
This removes all test coverage for `create` method.

Reply:
I'm still reading through the diffs I haven't done a final pass on this yet.

Reply:
OK, diff reviewed.